src/api/migrations/index.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import { join as joinPath } from 'path';
+
+import { sync as syncGlob } from 'glob';
+import { moveSync } from 'fs-extra';
+
+import type { MigratorType } from '../../types';
+
+const migrations = ([
+  (config) =>
```

Comment:
I believe it's actually not hidden
`migrations` is an array of migration functions – there happens to be a single one in here right now, which is why I did not split it off **yet**